web-client/package.json

```diff
@@ -40,9 +40,9 @@
   },
   "scripts": {
     "start": "SKIP_PREFLIGHT_CHECK=true react-app-rewired start",
-    "build": "react-app-rewired build",
-    "test": "react-app-rewired test",
-    "eject": "react-app-rewired eject",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject",
```

Comment:
We were using react-app-rewired to allow for theme customization here. Is it possible to maintain it that way?

Reply:
Oh shoot :( Yes try switching it back and running `npm run build`

Reply:
I'll look into this to see if I can resolve, using rewired 😄